server/mscalendar/notification.go

```diff
@@ -124,7 +119,9 @@ func (processor *notificationProcessor) work() {
 func (processor *notificationProcessor) processNotification(n *remote.Notification) error {
 	sub, err := processor.Store.LoadSubscription(n.SubscriptionID)
 	if err != nil {
-		return err
+		// TODO: Only happening when the subs were removed locally but not on remote, I kept getting
+		// notifications even after removing the app manually from my google account page.
+		return nil
```

Comment:
For any todos related to gcal, I think having an indicator like "TODO gcal" would help us know where the todo came from, i.e. from the gcal migration. For instance, this could be a breaking change for mscalendar provider, though looks like risk is not too bad

Reply:
I had this only for local testing to avoid seeing error messages that didn't belong to what I was currenctly developing. I'm going to revert it to what it was.